### PR TITLE
Release/a11y fixes

### PR DESCRIPTION
**Pull request by ronaldmansveld**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by paul-graham — CHANGES_REQUESTED** [review]
I'm assuming the `package.json` will be updated to point at a fully released version of the toolkit before this gets merged. I've reviewed the other changes and I see no issues at all... but just to play it safe, I'm going to reserve my "Approve" until it's pointing at the released version. :)

File: src/app/app.module.ts [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
 import { CaseShareService } from './services/case/share-case.service';
+import { DefaultErrorHandler } from './services/errorHandler/defaultErrorHandler';
+import { AbstractAppInsights, AppInsightsWrapper } from './services/logger/appInsightsWrapper';
+import { CryptoWrapper } from './services/logger/cryptoWrapper';
+import { LoggerService } from './services/logger/logger.service';
+import { MonitoringService } from './services/logger/monitoring.service';
+import { SharedModule } from './shared/shared.module';
+import { effects } from './store/effects';
+// ngrx modules - END
+// APP store
```
Comment (paul-graham) [inline src/app/app.module.ts:34]: _Nitpick:_
This comment and the one above is no longer relevant, following the reordering of the imports.

**Review by paul-graham — DISMISSED** [review]
The `package.json` concern is no more. Just a couple of comments around the imports in `app.module.ts` that are no longer relevant but that's certainly not enough to stop approval. 👍

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by paul-graham — APPROVED** [review]

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by udaydenduluri33 — DISMISSED** [review]
LGTMm

**Review by rayliangatos — DISMISSED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by udaydenduluri33 — APPROVED** [review]
LGTMm

**Pull request merged**